package-lock.json

```diff
@@ -36331,7 +36333,7 @@
         "@web3-storage/db": "^4.0.0",
         "@web3-storage/multipart-parser": "^1.0.0",
         "@web3-storage/tools": "^1.0.0",
-        "@web3-storage/toucan-js": "*",
+        "@web3-storage/toucan-js": "^2.7.1",
```

Comment:
I can't tell if this is concerning or not.

Reply:
Probably just from pulling latest and installing deps?